### DISCUSSION TITLE
How to enable page scrolling on <canvas> element

### DISCUSSION THREAD

**Discussion by hanhanxue** [jeffreylanters/react-unity-webgl#520] [Help Wanted!]
Hello! So I noticed that when the cursor is over the <canvas> element scrolling is not working. How could I enable it? Especially when the canvas is full width it's not ideal to be stuck with using the scroll bar to navigate up / down the page.

Thank you

![e8f753d8a6aa674d68c262d0b540374ee192ca3a_2_1014x750](https://github.com/jeffreylanters/react-unity-webgl/assets/24802314/33db438f-d62f-4bea-95f6-90528e8222fc)

**Comment by jeffreylanters** [comment]
Hello! You can disable the canvas' scroll event using the `disabled canvas events` API.

The following example shows how you can disable scrolling event for the entire canvas.

```tsx
<Unity
  unityProvider={unityProvider}
  disabledCanvasEvents={["scroll"]}
/>
```

See the API reference for more information: https://react-unity-webgl.dev/docs/api/disabled-canvas-events.

Best of luck with your project!

**Reply by hanhanxue** [reply]
@jeffreylanters thank you for the quick reply!

I just read the comments inside the UnityArguments type definition and it's exactly what I'm looking for. However I added the `disabledCanvasEvents={["scroll"]}` line and it's still not letting me scroll in my project. Is there any way I can debug this on my end?

[Here is a page I'm using to test the behavior](https://www.supermanifold.com/process/react-unity-integration)

Thank you!

**Reply by jeffreylanters** [reply]
Thank you for your feedback, I see what's happening here. I'm going to try to look into adding this feature as soon as possible! I'll keep you posted.

**Reply by hanhanxue** [reply]
This is awesome and please do. Thank you!

**Reply by TravisChen** [reply]
@jeffreylanters Hi Jeffrey! Were you ever able to look into this?

**Reply by TravisChen** [reply]
@jeffreylanters @hanhanxue 

I ended up finding a solution that seems to work. 

Get a reference to the canvas and then add this. Effectively it disables pointer events while wheel scrolling or swiping. But enables them otherwise. 

https://pastebin.com/GbdNmcPX
